packages/modules/src/modules/banners/momentTemplate/settings.ts

```diff
@@ -0,0 +1,27 @@
+export type CtaStateSettings = {
+    textColour: string;
+    backgroundColour: string;
+    border?: string;
+};
+
+export interface CtaSettings {
+    default: CtaStateSettings;
+    hover: CtaStateSettings;
+}
+
+export interface ImageSettings {
```

Comment:
we were hoping to eventually support different breakpoints in the `Image` model [here](https://github.com/guardian/support-dotcom-components/blob/115498ae147c167fbb43817dd3c6a52f1d65a7f1/packages/shared/src/types/props/shared.ts#L134).
I'm not sure if it's worth combining them now?